src/lib/y2network/sysconfig/connection_config_writer.rb

```diff
@@ -36,6 +36,14 @@ def write(conn)
         file.save
       end
 
+      # Removes connection config from the underlying system
+      #
+      # @param name [String] Connection name to remove
+      def remove(name)
```

Comment:
np: I would call the method remove_ifcfg or something like that.

Reply:
well, it looks consistent for me with write. It is also `write` and not `write_ifcfg`